onadata/apps/api/viewsets/open_data_viewset.py

```diff
@@ -132,42 +132,34 @@ def data(self, request, **kwargs):
                 qs_kwargs.update({'id__gt': gt_id})
 
             instances = Instance.objects.filter(**qs_kwargs).order_by('pk')
-            length = self.total_count = instances.count()
 
             if count:
+                self.total_count = instances.count()
```

Comment:
do we even need to set a class variable here? Why not put `instances.count()` inline as the value below?